tests/phpunit/CRM/Custom/Import/Parser/ApiTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @file
+ * File for the CRM_Custom_Import_Parser_ContributionTest class.
+ */
+
+/**
+ *  Test Contribution import parser.
+ *
+ * @package CiviCRM
+ * @group headless
+ */
+class CRM_Custom_Import_Parser_ApiTest extends CiviUnitTestCase {
+
+  use CRMTraits_Custom_CustomDataTrait;
+  use CRMTraits_Import_ParserTrait;
+
+  /**
+   * Test the full form-flow import.
+   */
+  public function testImport(): void {
+    $this->individualCreate();
+    $this->createCustomGroupWithFieldOfType(['is_multiple' => TRUE, 'extends' => 'Contact'], 'select', 'level');
+    $customGroupID = $this->ids['CustomGroup']['level'];
+    $dateFieldID = $this->createDateCustomField(['date_format' => 'yy', 'custom_group_id' => $customGroupID])['id'];
+    $this->importCSV('custom_data_date_select.csv', [
```

Comment:
There's a test failure:

```
fopen(/home/jenkins/bknix-dfl/build/core-23736-3prig/web/sites/all/modules/civicrm/tests/phpunit/CRM/Custom/Import/Parser/data/custom_data_date_select.csv): Failed to open stream: No such file or directory
```

I think maybe this file wasn't committed?

Reply:
Nice change from the extra file commits @demeritcowboy has been culling for me :-)